custom-deploy-steps/create-task-definition/action.yml

```diff
@@ -94,5 +98,5 @@ runs:
       run: |
         aws ecs tag-resource \
           --resource-arn ${{ steps.deploy-task-definition.outputs.task-definition-arn }} \
-          --tags key=created_by,value='GitHub Actions Deployment' key=Name,value=${{ inputs.application_id }} \
+          --tags key=created_by,value=${{ inputs.deployment_tag }} key=Name,value=${{ inputs.application_id }} \
```

Comment:
The way you used the inputs is different here and above. Is there a good reason?

Reply:
No reason, I can revert to your preferred format

Reply:
Just wanted to make it consistent. For me both is fine, but I think we need to decide on a general pattern. I think I used a mixed way in the past as well.

Defining inputs / step outputs as variables could have the advantage, that the commands could be used on the local machine with just setting some env vars. On the other hand it could be confusing if the inputs are just used once.

So if you ask me, I would vote for defining the env vars.